packages/terra-action-header/translations/en-US.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
You can pull translations for the other terra supported locales from the terra-clinical-action-header: https://github.com/cerner/terra-clinical/tree/master/packages/terra-clinical-action-header/translations

We should have a translation json file for the following:
* ar.json
* de.json
* en-GB.json
* en-US.json
* en.json
* es-ES.json
* es-US.json
* es.json
* fi-FI.json
* fr-FR.json
* fr.json
* pt-BR.json
* pt.json

Reply:
Added translations [here](https://github.com/cerner/terra-core/commit/da3b29962d2458b4be17f1b392be310150eec267).